src/samplers/hmcda.jl

```diff
@@ -53,7 +53,7 @@ function HMCDA(n_iters::Int, n_adapts::Int, delta::Float64, lambda::Float64, spa
     return HMCDA(n_iters, n_adapts, delta, lambda, _space, 0)
 end
 
-function hmc_step(θ, lj, lj_func, grad_func, H_func, ϵ, alg::HMCDA, momentum_sampler::Function;
+function hmc_step(θ, lj, lj_func, grad_func, H_func, ϵ, alg::HMCDA, momentum_sampler;
```

Comment:
Why does changes on the compiler code change the `momentum_sampler`? I guess this should not be the case.

Reply:
You are probably right.